tools/scripts-core/scripts/build-lib-umd.js

```diff
@@ -1,19 +1,24 @@
-const spawn = require('cross-spawn');
-const { resolveBin, hereRelative } = require('../utils/path-resolver');
-const { getUserConfigFile } = require('../utils/env');
+/* eslint-disable import/extensions */
+import spawn from 'cross-spawn';
+import fs from 'fs';
+import { resolveBin, hereRelative } from '../utils/path-resolver.js';
+import { getUserConfigFile } from '../utils/env.js';
+import { getDirName } from '../utils/dirname.js';
 
 const webpack = resolveBin('webpack');
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const dirname = getDirName(import.meta.url);
 
 function buildUMD(env, presetApi, options = []) {
 	return spawn.sync(
 		webpack,
-		['--config', hereRelative(__dirname, '../config/webpack.config.js')].concat(options),
+		['--config', hereRelative(dirname, '../config/webpack.config.js')].concat(options),
 		{ stdio: 'inherit', env },
 	);
 }
 
-module.exports = function build(env, presetApi, options) {
-	const packageJSON = require(getUserConfigFile(['package.json']));
+export default function build(env, presetApi, options) {
+	const packageJSON = JSON.parse(fs.readFileSync(getUserConfigFile(['package.json'])));
```

Comment:
I usually do try catch around JSON.parse

Reply:
If I know what to do in the catch me too.
But in this case and others I prefer to have a stack trace and exit the process as something goes wrong. Which was the case also with require on JSON file